sdk/src/vote_transaction.rs

```diff
@@ -30,7 +31,11 @@ impl VoteTransaction {
         )
     }
 
-    pub fn new_account(
+    //TODO `vote` should be replaced with `stake` or `staking`
```

Comment:
staking

Reply:
I want to so this in a different pr maybe